datacontract/imports/dbt_importer.py

```diff
@@ -44,7 +46,9 @@ def read_dbt_manifest(manifest_path: str) -> Manifest:
     """Read a manifest from file."""
     with open(file=manifest_path, mode="r", encoding="utf-8") as f:
         manifest_dict: dict = json.load(f)
-    return Manifest.from_dict(manifest_dict)
+    manifest = Manifest.from_dict(manifest_dict)
+    manifest.build_parent_and_child_maps()
```

Comment:
We need to build child_map in advance for this line

- https://github.com/datacontract/datacontract-cli/pull/547/files#diff-b5d431da7ac31000c438134a0a2bc8d45257096a00e1144192579b244ca340d6R114